py4DSTEM/io/utils.py

```diff
@@ -0,0 +1,37 @@
+from os.path import splitext
+
+def parse_filetype(fp):
+    """ Accepts a path to a 4D-STEM dataset, and returns the file type.
+    """
+    _, fext = splitext(fp)
+    if fext in [
+        ".h5",
+        ".H5",
+        ".hdf5",
+        ".HDF5",
+        ".py4dstem",
+        ".py4DSTEM",
+        ".PY4DSTEM",
+        ".emd",
+        ".EMD",
```

Comment:
I don't think the accepted file extensions are described in any of the doc strings,  maybe raise in the exception what the expected types are?

Reply:
updated